src/rebar_prv_common_test.erl

```diff
@@ -118,6 +119,24 @@ format_error({error_reading_testspec, Reason}) ->
 %% Internal functions
 %% ===================================================================
 
+%% @doc Tries to make the symlink `_build_test_logs/last` to the `ct_run` directory
+%% of the last common test run.
+-spec symlink_to_last_ct_logs() -> ok.
+symlink_to_last_ct_logs() ->
+    LogsFolder = "_build/test/logs/",
```

Comment:
Will this work on Windows? `filename:join/1` joins a list of dirs.

Reply:
As per `filename:join/1` [documentation](https://erlang.org/doc/man/filename.html#join-1):

>The result is "normalized":
>- Redundant directory separators are removed.
>- In Windows, all directory separators are forward slashes and the drive letter is in lower case.

So I guess it would be fine?

Reply:
We do prefer to use filename:join in all cases to be safe.

Reply:
Another tricky thing: The logs folder may change based on the profiles; If you call `rebar3 as osx ct` you will get the directory `osx+ct`. You can use `rebar_dir:base_dir(State)` and it will return you the proper `/_build/<profile>/` directory, after which you can glue on `logs/`. So something like `LogDir = filename:join([rebar_dir:base_dir(State), logs)`